src/Symfony/Component/Yaml/Tests/InlineTest.php

```diff
@@ -507,17 +507,17 @@ public function testParseTimestampAsDateTimeObject($yaml, $year, $month, $day, $
         $expected = new \DateTime($yaml);
         $expected->setTimeZone(new \DateTimeZone('UTC'));
         $expected->setDate($year, $month, $day);
-        $expected->setTime($hour, $minute, $second);
+        @$expected->setTime($hour, $minute, $second, 1000000 * ($second - (int) $second));
```

Comment:
I had HHVM in mind, I don't know if the version check will be valid (now or later when they'll implement this). Do as you prefer :)